docs/OnboardingOutcomes.md

```diff
@@ -0,0 +1,112 @@
+---
+resourceType: Documentation
+personas: 
+  - Developer
+  - Product Owner
+  - Designer
+tags:
+  - openshift
+  - new
+  - onboarding
+  - project set
+  - devops
+description: Learn where to ask for help with issues not covered by devops-requests
+---
+
+# Onboarding Outcomes
+
+This document is primarily for the use of those teams who have already had an onboarding meeting with the Platform Services team,
+but it contains plenty of useful information for everyone else, too!
+
+## Our Expectations for You
+
+As one of the vanguard of openness and agility in government, your presence on the Openshift Platform comes with some expectations.
+At their core, these expectations boil down to one thing: **Leadership**. You're here to explore a new, more digital government.
+You're building the path that others will follow. That means that you will be expected to help others to follow that path.
+
+### A Community Supported Platform
+
+In short, this just means one thing: help each other. If you're having problems - with *anything* to do with being a DevOps team - your first stop for help should be the community.
+This refers to both the Developer Exchange community, and the larger development community on the web.
+
+When you're having a problem, your first stop should be to seek out the rest of the community. This probably means RocketChat (#devops-how-to is always a good place to start), and can also mean seeking out help from the community that exists around your chosen open-source tools on the wider web, as well.
+
+It also means that you should be keeping a close eye on rocketchat yourself, and offering support to other teams as they encounter problems. It's a two-way street!
+
+If you're from elsewhere in government, this is probably a process that is different from what you're used to - normally, you'd call 7-7000 and someone else would swing in to fix it for you.
+Self-service cuts both ways: you have all kinds of freedom and flexibility, but it also means responsibility. The more you provide your support to others, the more others will be able to support you.
+
+However, if you suspect a problem with *the platform itself* (such as an outage), head on over to [our status page](https://status.pathfinder.gov.bc.ca) or #devops-alerts and/or #general on RocketChat to see if we are already aware of a platform-wide problem. 
+If you don't see anything there, then #devops-operations is a good place to start. However, you should always be *sure* that it's a platform issue before posting there. If you're not sure, start on #devops-how-to to see if you can get some help with that.
+
+### An Open Platform
+
+Open is the name of the game. If it can be open, it should be. If it can't be open, you should aim to fix it so that it can.
+
+All of our teams' code is open-source and available on the bcgov github organization.
```

Comment:
```suggestion
All of our teams' code is open-source and available on the bcgov github organization with the exception of a few highly sensitive projects that for the time being keep their code in private repos until they figure out how to make it open to public in a safe manner.
```

Reply:
To me, this suggests exceptions rather than just apps that are on the path to becoming open. What about this:
```suggestion
All of our teams' code is either in open-source and available on the bcgov github organization, or is (in rare cases) in the process of adjusting their sensitive code in a manner that would allow them to move their repository to the open-source bcgov organization within the short or medium term.
```